# ping sender fill in this (pong sender copies)
uint32 timestamp1
# ping sets to 0 (pong sender fill this)
uint32 timestamp2
